apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/adresse/Norg2Klient.kt

```diff
@@ -30,7 +33,9 @@ class Norg2Klient(
                 return enhetCache
             }
 
-            val response = klient.get("$apiUrl/enhet/$enhet")
+            val response = klient.get("$apiUrl/enhet/$enhet") {
+                header(X_CORRELATION_ID, getXCorrelationId())
```

Comment:
Burde dette ligge høyere, altså på httpklienten? For den bruker den fra common-rest?

Reply:
Kan hende - men det har vi ikke gjort på alle andre. Så dette var quick fix.

Kanskje restModule burde gjøre dette her i alle tilfeller - men det kan være eget sak :)